app/controllers/users_controller.rb

```diff
@@ -0,0 +1,11 @@
+class UsersController < ApplicationController
+  
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
```

Comment:
You should probably have some error handling here incase you can't find that specific user.